<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PowerCall v2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2331</width>
      <height>1081</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Analisitab">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Analisi</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>401</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Workflow</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="alignment_check">
         <property name="text">
          <string>FASTQ Alignment</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="read_group_check">
         <property name="text">
          <string>Add or Replace Read Groups</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="markDup_check">
         <property name="text">
          <string>Marking Duplicates</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="indel_real_check">
         <property name="text">
          <string>Indel Realignement</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="BQSR_check">
         <property name="text">
          <string>Base Quality Score Recalibration</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>60</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="variant_call_check">
         <property name="text">
          <string>Variant Calling</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="variant_filter_check">
         <property name="text">
          <string>Variant Filtering</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="variant_ann_check">
         <property name="text">
          <string>Variant Annotation</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="CNV_check">
         <property name="text">
          <string>CNV Calling</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>401</width>
        <height>149</height>
       </rect>
      </property>
      <property name="title">
       <string>Analysis Informations</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Run ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Panel Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Illumina TrusighCardio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Illumina TrusighCancer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Illumina TrusighOne</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SophiaGen HCS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Agilent CardioV1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Agilent AneurysmV1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Agilent CancerV1</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>Germline</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Somatic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cellfree</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Pipeline</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>499</x>
        <y>30</y>
        <width>601</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string>Edit Sample sheet </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>601</width>
         <height>471</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Search: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
             <item>
              <widget class="QPushButton" name="SSBrowse_Button">
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="SSAccept_Button">
               <property name="text">
                <string>Save and Use</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>550</y>
        <width>1031</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>START POWERCALL</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Panelstab">
     <attribute name="title">
      <string>Panel settings</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Pipelinestab">
     <attribute name="title">
      <string>Pipelines Settings</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tools settings</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPowerCall_v2_0">
    <property name="title">
     <string>PowerCall v2.0</string>
    </property>
   </widget>
   <addaction name="menuPowerCall_v2_0"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SSBrowse_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SSBrowseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1067</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>1257</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SSAccept_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SSAcceptClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>994</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>1462</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SSBrowseClicked()</slot>
  <slot>SSAcceptClicked()</slot>
 </slots>
</ui>
